src/dict/mdx.cc

```diff
@@ -953,7 +954,8 @@ void MdxDictionary::replaceLinks( QString & id, QString & article )
         articleNewText += linkTxt;
         match = RX::Mdx::closeScriptTagRe.match( article, linkPos );
         if ( match.hasMatch() ) {
-          articleNewText += article.mid( linkPos, match.capturedEnd() - linkPos );
+          articleNewText += QString( QStringLiteral( "gdOnReady(()=>{%1});</script>" ) )
+                              .arg( article.mid( linkPos, match.capturedStart() - linkPos ) );
```

Comment:
I suspected that `article.mid( linkPos, match.capturedStart() - linkPos )` should already contain the original  `</script>`

the above line will resulted a line with double   `</script>`

Reply:
The `capturedStart` is the beginning of `</script>`, so it is not included.

Reply:
The 2nd pramater of `mid` is total number of chars.